bench/rocksdb/optimized/hash.cc.ll

```diff
@@ -398,17 +398,19 @@ for.body.i.i:                                     ; preds = %for.body.i.i, %if.t
   br i1 %exitcond.not.i.i, label %for.end.i.i, label %for.body.i.i, !llvm.loop !6
 
 for.end.i.i:                                      ; preds = %for.body.i.i
-  %conv.i194.i = trunc nuw i64 %n to i32
   %shr.i.i197.i = lshr i64 %add.i196.i, 37
   %xor.i32.i.i = xor i64 %shr.i.i197.i, %add.i196.i
   %mul.i.i198.i = mul i64 %xor.i32.i.i, 1609587929392839161
   %shr1.i.i.i = lshr i64 %mul.i.i198.i, 32
   %xor2.i.i.i = xor i64 %shr1.i.i.i, %mul.i.i198.i
-  %cmp73.i.i = icmp ugt i32 %conv.i194.i, 143
+  %cmp73.i.i = icmp ugt i64 %n, 143
   br i1 %cmp73.i.i, label %for.body8.preheader.i.i, label %_ZL22XXPH3_len_129to240_64bPKhmS0_mm.exit.i
 
 for.body8.preheader.i.i:                          ; preds = %for.end.i.i
-  %div29.i.i = lshr i64 %n, 4
+  %conv.i194.i = trunc nuw i64 %n to i32
+  %div29.i.i = lshr i32 %conv.i194.i, 4
+  %umax.i.i = tail call i32 @llvm.umax.i32(i32 %div29.i.i, i32 9)
+  %wide.trip.count.i.i = zext nneg i32 %umax.i.i to i64
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597406008